src/ngraph/ngraph_emitter.h

```diff
@@ -27,17 +27,18 @@ namespace ngraph_bridge {
 
 // Alias for maps of name, function, where function returns an ngraph node
 using OpEmitter =
-    std::map<std::string, std::function<NgraphNodePtr(const NodePtr&)> >;
+    std::map<std::string, std::function<NgraphNodePtr(const NodePtr&)>>;
 using GradientEmitter =
     std::map<std::string, std::function<NgraphNodePtr(const NodePtr&,
-                                                      const NgraphNodePtr&)> >;
```

Comment:
Still creeps me out that our code-formatter has more than one normal form for a given file.

Reply:
I added a line above, clang-format sometimes does odd things when you change other lines :)